File: app/src/main/java/com/example/android_kotlin_handson/model/Repository.java
package com.example.android_kotlin_handson.model;

public class Repository {
    public String fullName;

    public String htmlUrl;

    public Integer stargazersCount;

    public Repository(String fullName, String htmlUrl, Integer stargazersCount) {
        this.fullName = fullName;
        this.htmlUrl = htmlUrl;
        this.stargazersCount = stargazersCount;
    }
}

File: app/src/main/java/com/example/android_kotlin_handson/model/Repository.kt
package com.example.android_kotlin_handson.model

import com.google.gson.annotations.SerializedName

data class Repository(
        @field:SerializedName("full_name") val fullName: String,
        @field:SerializedName("html_url") val htmlUrl: String,
        @field:SerializedName("stargazers_count") val stargazersCount: Int
)
